app/scripts/controllers/edit/buildConfig.js

```diff
@@ -312,16 +335,18 @@ angular.module('openshiftConsole')
             // If we found the item successfully, watch for changes on it
             watches.push(DataService.watchObject("buildconfigs", $routeParams.buildconfig, context, function(buildConfig, action) {
               if (action === 'MODIFIED') {
-                $scope.alerts["updated/deleted"] = {
+                NotificationsService.addNotification({
+                  id: 'edit-build-config-conflict',
```

Comment:
you added ids for these two warnings, did you intend to clean them up in hideErrorNotifications?